corehq/apps/app_manager/templates/app_manager/partials/releases.html

```diff
@@ -85,7 +85,7 @@
 {% include 'app_manager/partials/download_async_modal.html' with element_id='download-zip-modal' %}
 
 
- {% if is_mobile_experience %}
+ {% if request.user_agent.is_mobile %}
```

Comment:
Should this also be restricted to users who are on the newer side? I don't think this will change behavior for anyone who has been around for awhile

Reply:
i think its fine to end up in mobile and see the pop up once.

Reply:
ok